README.md

```diff
@@ -3,5 +3,9 @@
 # elastic-serverless-forwarder
 Elastic Serverless Forwarder
 
+## Important - v1.6.0
 
+#### Version 1.6.0 introduces a new way to generate event IDs. More information is available at [our troubleshooting documentation](https://www.elastic.co/guide/en/observability/master/aws-serverless-troubleshooting.html).
```

Comment:
this suggests to  me there there is some configuration available to generate new IDs. i would rephrase with something more definite e.g. "introduces a new event ID format which is backwards incompatible with previously indexed events; be aware that previously indexed events would be duplicated if they triggered the forwarder again after upgrading to this version".

Reply:
Good point 👍 

I already mentioned your last point in the troubleshooting documentation, but I agree that is best re-iterating it here, as it's the first thing users would see.